projects/plugins/protect/src/js/components/threats-list/styles.module.scss

```
@@ -99,3 +99,44 @@
 		}
 	}
 }
+
+.pagination-container {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	margin-top: calc( var( --spacing-base ) * 2 ); // 16px
+
+	button {
+		font-size: var( --font-body );
+		padding: var( --spacing-base ) calc( var( --spacing-base ) * 2 ); // 8px | 16px
+
+		&.unfocused {
+			color: var( --jp-black );
+			background: none;
+			box-shadow: none;
+	
+			&:hover:not(:disabled), &:focus:not(:disabled) {
+				color: var( --jp-black );
+				background: none;
+				box-shadow: none;
+			}
+		}
+
+		&.icon-button:focus:not(:disabled)  {
+			box-shadow: none;
+		}
+	}
```

```suggestion

        > svg {
            width: 1.5rem; // 24px
            height: 1.5rem; // 24px
        }
	}
```
Suggestion: smaller icon sizes to match design.

---

Addressed via [34e9ee8](https://github.com/Automattic/jetpack/pull/39058/commits/34e9ee8a74bd0d7b13247fba544b8ca44103f8b0)